CMakeLists.txt

```diff
@@ -51,6 +51,10 @@ target_link_libraries(${PROJECT_NAME}
 	protobuf::libprotobuf 
 	spdlog::spdlog)
 
+configure_file(${CMAKE_SOURCE_DIR}/valgrind_exclude_tests.txt
+			${CMAKE_BINARY_DIR}/valgrind_exclude_tests.txt
+			COPYONLY)
+
```

Comment:
If the exclude list moves, this could move into the `CMakeLists.txt` file in the `test/` directory.

Reply:
Thanks, that's great, it will make changes more localized. I'll move it to the file in test dir